dot-gen/src/main/java/com/wipro/ats/bdre/ldg/GetDotForTable.java

```diff
@@ -0,0 +1,132 @@
+             }
+
+            //this should check whether the nodes are target -- ??
+            lineageRelationListForTargetNodes = getNodeIdForLineageRelation.getWhenTargetNode(lineageNode.getNodeId());
+            for(LineageRelation lineageRelation:lineageRelationListForTargetNodes) {
+                //if (lineageRelationList.size() != 0) {
+                logger.info("------------------------inside target check");
+                relationDot.append("\n" + lineageRelation.getDotString() + "\n");                                            //search for target table name in the DOT string
+                logger.info("------------------" + relationDot);
+                targetTableNode.append("\n" + getLineageNodeByColName.getTableDotFromNodeId(lineageRelation.getLineageNodeBySrcNodeId(), srcTableNodeList.get(0)) + "\n");
+                logger.info("-------------------------" + targetTableNode);
+                if(targetTableNode.equals("same-nodes")) {
+                    targetTableNode.append("\n" + getLineageNodeByColName.getTableDotFromNodeId(lineageRelation.getLineageNodeByTargetNodeId(), srcTableNodeList.get(0)) + "\n");
+                }
+                logger.info("-------------------------------relation DOT: \n" + relationDot);
+            }
+
+        }
+        dotString = lineageMain.generateLineageDot(relationDot.toString(), srcTableNodeList.get(0).getDotString(), targetTableNode.toString());             //call lineage main to create the DOT
+        return dotString;
+    }
+
+    //fn() to get DOT through only Table name
+    public String dotGeneratorWithTable(String args[]) {
+        if(args.length !=1) {
+            logger.info("One argument required: please Give Table name");
+            System.exit(-1);
+        }
+        GetLineageNodeByColName getLineageNodeByColName = new GetLineageNodeByColName();;
+        LineageMain lineageMain = new LineageMain();
+        String srcTableName = args[0];
+        logger.info("Generating DOT for the given Table Name: " + args[0]);
+        String targetTableName = null;
+        String relationDot = null;
+        String targetTableNode = null;
+        List<LineageNode> srcTableNodeList = getLineageNodeByColName.getTableDotFromTableName(srcTableName);
+        dotString = lineageMain.generateLineageDot(relationDot, srcTableNodeList.get(0).getDotString(), targetTableNode);             //call lineage main to create the DOT
+        return dotString;
+    }
+
+    private LineageNode getLineageNodeNodeId (String col, LineageNode tableNode) {
+        GetLineageNodeByColName getLineageNodeByColName = new GetLineageNodeByColName();
+        LineageNode lineageNode = getLineageNodeByColName.execute(col, tableNode);
+        //for(LineageNode lineageNode:lineageNodeList){
+            logger.info("Column name: "+lineageNode.getDisplayName()+" Node id: "+lineageNode.getNodeId());
+        //}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)